### ISSUE TITLE
tour is not screen reader accessible

### ISSUE THREAD

**Issue by derekriemer** [golang/tour#1437]
Context: https://go.dev/tour/welcome/1

The go playground and tour of go use codemirror, which as of version 5 was not accessible. Codemirror 6 is accessible, but codemirror5 isn't. Please either allow blind developers to disable code mirror and use a textarea, switch to monico, or switch to codemirror6.

**Comment by zersiax** [comment]
To expand on this:
- CM5 is both rather old and not accessible to screen reader users, in the sense that screen reader users can't see what they're typing, nor arrow through code that is already there necessitating copying the code out into another window, making any modifications required and finally pasting it back. This is very inefficient and used to be a common barrier to entry to websites like CodeCademy, FreeCodeCamp and various other large and smaller-scale online course providers and coding playgrounds.
- A lot of work was put into CM6 to fix this, and this is also the current version receiving upgrades. If sticking with CM, upgrading makes sense from a futureproofing perspective.
- Monaco is Vs Code's editor component which a lot of the above-mentioned websites now use, as it filled a vacuum in the abscence of an accessible CodeMirror alternative. It would similarly fix this issue if the tour were to use Monaco instead.

As it stands now, I as a screen reader user would not want to use the tour given its accessibility issues.
